custom_components/miio2/manifest.json

```diff
@@ -4,8 +4,8 @@
   "version": "2022.3.3",
   "documentation": "https://github.com/KrzysztofHajdamowicz/home-assistant-vacuum-styj02ym",
   "requirements": [
-    "construct==2.10.56",
-    "python-miio==0.5.12"
+    "construct",
```

Comment:
It's a bad idea to remove versions of dependencies. Just update the construct to 2.10.70.